agent/engine/task_manager.go

```diff
@@ -434,7 +434,7 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 	container := containerChange.container
 	runtimeID := container.GetRuntimeID()
 	event := containerChange.event
-	logger.Info("Handling container change event", logger.Fields{
+	logger.Debug("Handling container change event", logger.Fields{
```

Comment:
if logs from customers are just at info level, is there a way to find out if a docker event was received for the container state change?

Reply:
I was thinking about this one and I am a bit thorn. On the one hand, this log shows the events that the agent receives from docker, which is useful when it's the first time the event occurs (e.g. `PULLED`, `CREATED`), but it quickly gets noisy because docker keeps sending events for running containers. 

Maybe a good compromise would be to display this log in info level but only when `GetKnownStatus != event.Status`?

Reply:
I like that!